pyppeteer/page.py

```diff
@@ -226,7 +277,11 @@ def browser(self) -> 'Browser':
         """Get the browser the page belongs to."""
         return self._target.browser
 
-    def _onTargetCrashed(self, *args: Any, **kwargs: Any) -> None:
+    @property
+    def browserContext(self):
```

Comment:
You'll need to import from browser (guard with TYPE_CHECKING)
```suggestion
    def browserContext(self) -> BrowserContext:
```